src/app/pages/LendingPage/components/LendingDialog/index.tsx

```diff
@@ -156,6 +153,11 @@ export function LendingDialog({
       : [tokenAddress, withdrawAmount];
   }, [currency, tokenAddress, type, weiAmount, withdrawAmount]);
 
+  const handleSubmit = () => {
```

Comment:
Just use a ternary expression and rewrite it to `const handleSubmit = () =>  type === 'add' ? handleLendSubmit() : handleUnlendSubmit();` .

Also, use `useCallback`.